diff --git a/node_modules/@bam.tech/react-native-image-resizer/react-native-image-resizer.podspec b/node_modules/@bam.tech/react-native-image-resizer/react-native-image-resizer.podspec
index c7ba7d9..7498032 100644
--- a/node_modules/@bam.tech/react-native-image-resizer/react-native-image-resizer.podspec
+++ b/node_modules/@bam.tech/react-native-image-resizer/react-native-image-resizer.podspec
@@ -32,6 +32,7 @@ Pod::Spec.new do |s|
     s.dependency "RCTRequired"
     s.dependency "RCTTypeSafety"
     s.dependency "ReactCommon/turbomodule/core"
+    install_modules_dependencies(s)
   end
 end
 
